artifactory/commands/buildinfo/publish.go

```diff
@@ -229,3 +236,15 @@ func (bpc *BuildPublishCommand) getNextBuildNumber(buildName string, servicesMan
 	latestBuildNumber++
 	return strconv.Itoa(latestBuildNumber), nil
 }
+
+func recordJobSummaryData(buildInfo *buildinfo.BuildInfo, buildLink string) (err error) {
```

Comment:
recordJobSummaryData --> recordSummaryData

Let's a clear description for this function, allowing random readers who know nothing about this functinality to understand it.